docs/pages/reference/machine-id/diagnostics-service.mdx

```diff
@@ -132,3 +117,67 @@ performance issues.
 
 They will only be enabled if the `-d`/`--debug` flag is provided when starting
 `tbot`. This is known as **debug mode**.
+
+## Prometheus Metrics
+
+The `tbot` process exposes a number of Prometheus metrics via the `/metrics`
+endpoint of the diagnostics service.
+
+In addition to exporting the standard Go runtime metrics, `tbot` also exports
+custom metrics that reflect the health and performance of the various
+configurable services.
+
+## Advice
+
+When monitoring the health of `tbot`, there are largely three categories of
```

Comment:
Is "largely" needed here? If it is, I would make it explicit when any exceptions apply.

Reply:
[c57281e](https://github.com/gravitational/teleport/pull/58970/commits/c57281e4c8949ae49e42cd39fcea2ddd75261879)